crates/exex/exex/src/manager.rs

```diff
@@ -530,6 +533,11 @@ pub struct ExExManager {
     /// The finished height of all `ExEx`'s.
     finished_height: watch::Sender<FinishedExExHeight>,
 
+    /// Write-Ahead Log for the [`ExExNotification`]s.
+    wal: Wal,
```

Comment:
we should def document why this is even needed.

Reply:
https://github.com/paradigmxyz/reth/issues/11201